lib/statistics/distribution/normal.rb

```diff
@@ -79,5 +79,59 @@ def density_function(value)
         euler/Math.sqrt(2 * Math::PI)
       end
     end
+
+    # Standard Normal Inverse distribution:
+    # References:
+    # https://en.wikipedia.org/wiki/Inverse_distribution
+    # http://www.source-code.biz/snippets/vbasic/9.htm
+    class StandardNormalInverse < StandardNormal
+      A1 = -39.6968302866538
```

Comment:
can you add a comment that explains that all this constants are rational coefficients used for approximation?

Reply:
I fixed all of those, but here I'd like to do so but I can't, because mainly I back-ported visual basic code to Ruby language

https://stackedboxes.org/2017/05/01/acklams-normal-quantile-function/

see this discussion